include/zephyr/drivers/cellular.h

```diff
@@ -52,6 +52,37 @@ struct cellular_network {
 	uint16_t size;
 };
 
+enum cellular_signal_type {
+	CELLULAR_SIGNAL_RSSI,
+	CELLULAR_SIGNAL_QUALITY,
+	CELLULAR_SIGNAL_PERCENTAGE,
+};
+
+enum cellular_signal_quality {
+	CELLULAR_MODEM_SIGNAL_EXCELLENT,
+	CELLULAR_MODEM_SIGNAL_GOOD,
+	CELLULAR_MODEM_SIGNAL_BAD,
+};
```

Comment:
I think we should skip this for now, and make it part of an inline helper `cellular_quality_from_rssi()` or similar

Reply:
Another point is whether there are widely acceptable categorization thresholds to say whether the signal is "excellent", "good", or "bad". Maybe the application wants to define those?

Reply:
I'm not opinionated so will remove them for now